src/platform/Ameba/Logging.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <platform_stdlib.h>
+
+namespace chip {
+namespace Logging {
+namespace Platform {
+
+enum AmebaLogLevel
```

Comment:
consider using `enum class'.

Constants names should be `kCamelCase`: `kError`, `kProgress` and such.
or if not enum class, then `inline constexpr uint8_t kAmebaLogLevelError = 0` (since then there seems no point to enum).

Reply:
So ideally we should have :

```
enum class AmebaLogLevel {...}

void LogSetLevel(AmebaLogLevel level);
AmebaLogLevel GetLogLevel();
```

also wondering if `chip::Logging::Platform` is the right namespace since things seem to be named Ameba always. Maybe `chip::Logging::Ameba` is the correct placement.

Reply:
Thanks for the suggestions, changed to use enum class and also changed namespace to Ameba